test/apps/rails4/lib/sweet_lib.rb

```diff
@@ -2,4 +2,9 @@ class SweetLib
   def do_some_cool_stuff bad
     `ls #{bad}`
   end
+
+  def test_find_group
+    #Should warn, no escaping done for :group
```

Comment:
Does this make you feel better? https://github.com/presidentbeef/brakeman/pull/610